certora/funnels/DepositorUniV3.spec

```diff
@@ -609,9 +574,6 @@ rule collect(DepositorUniV3.CollectParams p) {
     address buffer = buffer();
     require buffer != poolCon;
 
-    mathint wardsBefore = wards(anyAddr);
```

Comment:
The  `anyAddr`, `anyAddr_2` and `anyUint24` vars above are no longer used.

Reply:
Done!